### PR TITLE
feat: add chart card

### PR DESCRIPTION
**Pull request by jessejanderson**
https://github.com/geolessel/pco-dashboard/issues/15

This is incomplete - currently it renders a chart but it disappears nearly instantly. I'm assuming this is an issue with LiveView.

Also, currently the chart details for c3 are currently magic numbers defined inside of `app.js`. This will need to be generated with `chart-card.html.leex` somehow.

### CODE REVIEW

**Comment by jessejanderson** [comment]
Also, since the chart disappears instantly, there aren't any styles applied currently. :)

**Comment by geolessel** [comment]
The data is still fake, but we've got a render!

Screen Shot 2020-05-06 at 11 54 21 AM

Let me know if you need any more help getting the real data in there.

**Comment by geolessel** [comment]
Now I've got one rendering from API calls. I had to abandon headcounts as I couldn't figure out a good API call to make to get the right information. I know People's API well, so I did a people created over time chart.

Screen Shot 2020-05-06 at 4 12 17 PM

**Comment by jessejanderson** [comment]
@geolessel I think this is ready now other than the broken headcounts_chart.

File: apps/dashboard_web/assets/js/renderC3Chart.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        x: {
+          show: true
+        },
+        y: {
+          show: true
+        },
+      },
+    })
+
+    window.chart = this.chart
```
Comment (geolessel) [inline apps/dashboard_web/assets/js/renderC3Chart.js:36]:
```suggestion
```

File: apps/dashboard_web/lib/dashboard_web/live/components/profiles_created_over_time_chart.ex [hunk 2]
```diff
@@ -0,0 +1,35 @@
+defmodule DashboardWeb.Components.ProfilesCreatedOverTimeChart do
+  use DashboardWeb, :live_component
+  @behaviour DashboardWeb.Behaviours.ComponentLiveView
+
+  @impl true
+  def mount(socket) do
+    {:ok, assign(socket, :data, %{})}
```
Comment (geolessel) [inline apps/dashboard_web/lib/dashboard_web/live/components/profiles_created_over_time_chart.ex:7]:
```suggestion
    {:ok, assign(socket, :columns, [])}
```

**Pull request merged**